From c67f214546a9d8ac00b9b947c145f4c032def8e2 Mon Sep 17 00:00:00 2001
From: Yunhui Cui <yunhui.cui@nxp.com>
Date: Mon, 16 May 2016 14:39:52 +0800
Subject: [PATCH 32/93] driver: spi: fsl_qspi: enable AHB read for qspi

If we don't enable the AHB read for ls1012a, input 'md 0x40000000',
'md 0x41000000','md 0x42000000' address will be overlapped.
After QSPI controller initialization for AHB, 'md 0x...' will access
the whole QSPI flash address space.

Signed-off-by: Yunhui Cui <yunhui.cui@nxp.com>
---
 include/configs/ls1012a_common.h |    1 +
 1 file changed, 1 insertion(+)

diff --git a/include/configs/ls1012a_common.h b/include/configs/ls1012a_common.h
index 150444d..121824c 100644
--- a/include/configs/ls1012a_common.h
+++ b/include/configs/ls1012a_common.h
@@ -66,6 +66,7 @@
 
 #define FSL_QSPI_FLASH_SIZE		(1 << 26)
 #define FSL_QSPI_FLASH_NUM		2
+#define CONFIG_SYS_FSL_QSPI_AHB
 
 /*
  * Environment
-- 
1.7.9.5

